src/CliptoExchange.sol

```diff
@@ -2,11 +2,13 @@
 pragma solidity 0.8.10;
 
 import {CliptoToken} from "./CliptoToken.sol";
+import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
+import {ReentrancyGuard} from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
```

Comment:
If we're going to use reentrancy guard then we could use the Rari implementation which is gas optimized.

```
import {ReentrancyGuard} from "../lib/solmate/src/utils/ReentrancyGuard.sol";
```

Reply:
I switched it, saving <_drumroll_> **3** gas.

Reply:
Didn't realize it was this low, my bad :D